### DISCUSSION TITLE
Add Energy recovery ventilator

### DISCUSSION THREAD

**Discussion by alessioc01** [OlivierZal/com.melcloud#209] [Ideas]
May I ask you if you can plan to include the management of the LOSSNAY VMC Energy recovery ventilator
With the simple functions that the app allows:

- ON-OFF
- Speed ​​1-2-3-4
- Mode: Heat recovery-Bypass-Auto
- Outside air temperature

I can offer myself as a tester if you need it

Thank you

I would like to take this opportunity to thank you for the app, I use it a lot to manage summer temperatures throughout the house

**Comment by OlivierZal** [comment]
Hi @alessioc01, thanks for your feedback!

The only question is: do you see this device on the official [MELCloud (mobile or web) app](https://app.melcloud.com)?

If so, from this MELCloud app can you provide olivier.zalmanski@me.com with guest access so I can study the feasibility?

Kind regards

**Reply by OlivierZal** [reply]
Hi @alessioc01, were you able to check whether the device is on MELCloud?

**Reply by alessioc01** [reply]
Sorry, I didn't see your reply.

I attach what I see, and invite you as a guest with your email

![image](https://github.com/OlivierZal/com.melcloud/assets/58922250/bb17ca78-6429-4599-8197-aa3c78369b05)

**Reply by OlivierZal** [reply]
Thanks @alessioc01 for the access, I'll need some time to study the API behavior, and of course to perform some tests – with your agreement, I'll tell you beforehand but I will always restore your settings in the very next few seconds.

I'll start next week when I come home. 

In order to ease our communication, I invite you to join Homey's Slack community: https://slack.athom.com/

You seem to be an Italian, I'll try to translate the app in the same move.
